src/main/java/Counter.java

```diff
@@ -0,0 +1,11 @@
+public class Counter {
+    private int count;
+
+    public int getCount() {
+        return count;
+    }
+
+    public void increment() {
+        count++;
```

Comment:
```suggestion
    public int increment() {
      return  count++;
```
that'll make the code shorter